src/logion/model/locrequest.model.ts

```diff
@@ -248,7 +256,7 @@ export class LocRequestAggregateRoot {
     }
 
     addLink(itemDescription: LinkDescription) {
-        this.ensureOpen();
+        this.ensureOpen(true);
```

Comment:
This should fail because called by controller, no?

Reply:
It's called only during synchro.